sdk/docs/storage/README.md

```diff
@@ -5,49 +5,67 @@ Azure Storage Blobs library (`az_storage_blobs`) provides abstractions, and help
 Use the Azure SDK Storage Blobs Library for Embedded C to work with Azure Storage Blobs.
 
 * Upload blobs
-* Create and modify blob metadata
 
 ## Getting Started
 
-### Install the Package
+The Azure Storage Blobs Client library is created to facilitate connectivity to Azure Storage Blobs services using the HTTP stack of the user's choice. This means that this SDK is **NOT** a platform but instead is a true SDK library.
```

Comment:
I don't understand what we are trying to say with that last sentence. Seems odd to bring up.

```suggestion
The Azure Storage Blobs Client library is created to facilitate connectivity to Azure Storage Blobs services using the HTTP stack of the user's choice.
```

Reply:
It helps the user understand that they can control the underlying behavior.  That is distinct difference from the previous C SDKs which had implementations which could not be changed

Reply:
Makes sense. I don't know how to word that well, but using overloaded terms like "it is not a platform", and "true SDK library" are confusing. I start asking these questions: What's a platform in this context? What's a true SDK library supposed to be?

Maybe...
```suggestion
The Azure Storage Blobs Client library is created to facilitate connectivity to Azure Storage Blobs services using the HTTP stack of the user's choice. This capability differentiates the Embedded SDK from the previous C SDKs which had underlying implementations that the user could not be changed.
```

> underlying transport (platform) 

Also, we have been a few conversations (with @JeffreyRichter) about terminology around platform, and HTTP transport. Let's make sure to keep those two distinct (i.e. platform != http transport adapter)
See this issue: https://github.com/Azure/azure-sdk-for-c/issues/935

Reply:
I suggest this:
The Azure Storage Blobs Client library facilitates connectivity to the Azure Storage Blobs service. Since this service uses HTTP, you must plug in an HTTP adapter capable of sending the HTTP request to your device's HTTP stack. For information on how to do this, please see (link goes here).

(I'm assuming that we'll have more HTTP clients in the future and so the HTTP adapter stuff should be in 1 place and then simply referred to by the various clients)